src/uwtools/tests/drivers/test_driver.py

```diff
@@ -57,34 +57,37 @@ def write(path, s):
 
 
 @fixture
-def driverobj(tmp_path):
-    cf = write(
```

Comment:
This doesn't need to be written to a file and read back in, it can be used as an in-memory object.